debezium-connector-mongodb/src/main/java/io/debezium/connector/mongodb/MongoDbConnectorConfig.java

```diff
@@ -334,6 +334,15 @@ public static SnapshotMode parse(String value, String defaultValue) {
             SNAPSHOT_MODE, CommonConnectorConfig.SOURCE_STRUCT_MAKER_VERSION,
             Heartbeat.HEARTBEAT_INTERVAL, Heartbeat.HEARTBEAT_TOPICS_PREFIX);
 
+    /** The comma-separated list of operations in MongoDB OpLog  that reader will skip reading them from OpLog */
+    public static final Field SKIP_OPLOG_OPEARATIONS = Field.create("skip.oplog.operations")
```

Comment:
Ah, I was actually thinking that this could be done generically for all connectors. I.e. move the property to `CommonConnectorConfig` and rename it so it's not specific to MongoDB. I'll leave it to you whether you'd want to make use of it for other connectors, too, as part of this PR or not. Totally fine to limit to MongoDB initially, but foundations should be made so it can be rolled out for other connectors down the road.

Reply:
Simply I moved it on `CommonConnectorConfig` and renamed it to `skip.snapshot.operations`